openreview/openreview.py

```diff
@@ -276,10 +280,11 @@ def post_group(self, group, overwrite = True):
         """
         if overwrite or not self.exists(group.id):
             if not group.signatures: group.signatures = [self.signature]
+            if not group.writers: group.writers = [self.signature]
             response = requests.post(self.groups_url, json=group.to_json(), headers=self.headers)
             response = self.__handle_response(response)
 
-        return Group.from_json(response.json())
+        return self.get_group(response.json()['id'])
```

Comment:
one way or another, I think the post_group function should return the whole, up-to-date group object. If changing the API is a better solution than doing another get, then that's fine